itou/utils/perms/context_processors.py

```diff
@@ -27,30 +30,52 @@ def get_current_organization_and_perms(request):
                 request.user.siaemembership_set.select_related("siae").filter(siae__pk__in=user_siae_set_pks).all()
             )
             user_siaes = [membership.siae for membership in memberships]
-
-            for membership in memberships:
+            first_siae_idx = 0
+            for idx, membership in enumerate(memberships):
                 if membership.siae_id == siae_pk:
                     siae = membership.siae
                     user_is_siae_admin = membership.is_siae_admin
+                    first_siae_idx = idx
                     break
             if siae is None:
                 if request.path != reverse("account_logout"):
                     raise PermissionDenied
 
+            if first_siae_idx != 0:
+                user_siaes[0], user_siaes[first_siae_idx] = user_siaes[first_siae_idx], user_siaes[0]
+
+        # Prescriber organization ?
         prescriber_org_pk = request.session.get(settings.ITOU_SESSION_CURRENT_PRESCRIBER_ORG_KEY)
+
         if prescriber_org_pk:
-            membership = request.user.prescribermembership_set.select_related("organization").get(
-                organization_id=prescriber_org_pk
-            )
-            prescriber_organization = membership.organization
-            user_is_prescriber_org_admin = membership.is_admin
+            first_prescriber_idx = 0
+            memberships = current_user.prescribermembership_set.select_related("organization")
+
+            for idx, membership in enumerate(memberships):
+                # Same as above:
+                # In order to avoid an extra SQL query, fetch related organizations
+                # and artifially reconstruct the list of organizations the user belongs to
+                # (and other stuff while at it)
+                user_prescriberorganizations.append(membership.organization)
+                if membership.organization.pk == prescriber_org_pk:
+                    prescriber_organization = membership.organization
+                    user_is_prescriber_org_admin = membership.is_admin
+                    first_prescriber_idx = idx
+
+            if first_prescriber_idx != 0:
+                # Put selected prescriber organization first (UI)
```

Comment:
Je suis pas certain de la plus-value de mettre l'org courante toujours en haut de la liste.

Check sur Github par exemple, les items restent dans le même ordre peu importe vers quelle org tu bascules :

![orgs](https://user-images.githubusercontent.com/281139/96011405-29187b00-0e43-11eb-9380-eef2d144cc81.png)

Reply:
@celine-m-s @vgrange Un avis contradictoire rapide (avant de merger) ?

Reply:
Ca me va comme ça.

Reply:
"Simple is better than complex."